### ISSUE TITLE
Change prism.js theme?

### ISSUE THREAD

**Issue by ghost** [greglobinski/gatsby-starter-hero-blog#91]
I'd like to change the theme of the prism code blocks. I don't think a yaml file is appropriate for this. Can I just import a css file and use it that way?

**Comment by ghost** [comment]
Closing this I got it.

**Comment by ryexley** [comment]
@twhite96 can I ask how you changed it? I have a theme set, but the theme doesn't render the same between the `gatsby develop` and `gatsby build` tasks. I'm wondering if I've got it wrong somewhere. I initially changed it where it was imported in `src/templates/PostTemplate.js`, and that didn't work, then I found [this article](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-remark-prismjs) which said it should be required in `gatsby-browser.js`, and that doesn't seem to have worked either (so I removed it).

This is what it looks like locally when I'm running `gatsby develop`:

![image](https://user-images.githubusercontent.com/173844/73659082-84169300-4663-11ea-9f3d-997562223f26.png)

But this is what I see when it's been deployed and built with `gatsby build`:

![image](https://user-images.githubusercontent.com/173844/73659150-a01a3480-4663-11ea-86e9-15d3541f6459.png)

I'm not sure what I'm doing wrong.
